embedly/embedly/api/views.py

```diff
@@ -43,3 +43,14 @@ def extract_urls():
         status=200,
         mimetype='application/json',
     )
+
+
+@blueprint.route('/v2/extract', methods=['POST'])
+def extract_urls_v2():
+    urls = json.loads(request.values.get('urls', '[]'))
```

Comment:
the try catch block can catch other exceptions, coming from `extractor.extract_urls` and does `return "", 500` in that case.

So, something like:

```python
try:
    urls = json.loads(request.values.get('urls', '[]'))
    # or alternatively, urls = request.json or [] but request.json might not throw an exception
except TypeError as e:
    return "", 400
except:
    return "", 500
```